[DPE-7242] Add multi-cluster refresh docs (#690)
canonical/mysql-operator: docs/how-to/cluster-cluster-replication/index.md

# Cluster-cluster replication

Cluster-cluster asynchronous replication focuses on disaster recovery by distributing data across different servers.

> For increased safety, it is recommended to distribute data across different geographical regions as well.

Reviewer: I think using the terms _deploy cluster_ instead of _distribute data_ makes it closer from the user perspective.

```suggestion
For increased safety, it is recommended to deploy each cluster in a different geographical region.
```
